Header header
string[] addresses
float32[] flowrates
